<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SentSandPage</class>
 <widget class="QWidget" name="SentSandPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
background-color:rgb(51,153,255);;
font:12px &quot;SimHei&quot;;
color:white;
}

QPushButton
{
outline:none;
border-image: url(:/image/buttonUP.jpg);
font:22px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;
}</string>
  </property>
  <widget class="QPushButton" name="btnTest">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>测试
开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHeat">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>加热
开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>285</y>
     <width>80</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTinfo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16px;
color:red;
</string>
   </property>
   <property name="text">
    <string>测试中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelHinfo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16px;
color:red;
</string>
   </property>
   <property name="text">
    <string>加热中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
